checkers_bot_tournament/board.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, board_start_builder: BoardStartBuilder, size: int = 8):
 
         self.grid: Grid = board_start_builder.build()
 
-    def move_piece(self, move: Move) -> Tuple[bool, bool]:
+    def move_piece(self, move: Move) -> Tuple[int, bool]:
```

Comment:
Not something to change rn, but I think I should've made position a dataclass instead of just a tuple, and similarly I think ideally this should also return a type (i.e. dataclass) with named values.

Reply:
could a callback each for capture and promotion work?

Reply:
also yeah slightly annoying to remember which way is col and row but see: https://youtu.be/vCLetdhswMg?si=VjFI0kTPDnEizVyR&t=426 for performances of anything you can think of